.github/workflows/smoke_test.yml

```diff
@@ -37,10 +58,13 @@ jobs:
       - name: set environment (workflow dispatch)
         if: ${{ github.event_name == 'workflow_dispatch' }}
         run: |
-          echo "PAAS_ENVIRONMENT=${{ github.event.inputs.paas_environment }}" >> $GITHUB_ENV
+          echo "ENVIRONMENT=${{ github.event.inputs.paas_environment }}" >> $GITHUB_ENV
+
+      - name: Set up test database
+        run: bin/rails db:create db:schema:load
 
       - name: Run deployment smoke test
-        run:  bundle exec rspec spec/smoke_tests/participant_declarations_spec.rb --tag smoke_test
+        run: bundle exec rspec spec/smoke_tests/participant_declarations_spec.rb --tag smoke_test
 
       - name: print environment
-        run: echo ${{ env.PAAS_ENVIRONMENT}}
+        run: echo ${{ env.ENVIRONMENT}}
```

Comment:
Why?

Reply:
Cause `PAAS_ENVIRONMENT` is what Teaching Vacancies use, and `ENVIRONMENT` is what we use in our actions.

Reply:
I meant why do we echo it out here? Especially after the test step - that means we only print it if the test passes

Reply:
Moved it earlier